src/theme/components/breadcrumb.ts

```diff
@@ -14,6 +14,11 @@ const blogs = {
         color: "dark.ui.typography.heading",
       },
     },
+    _focus: {
+      outline: "none",
+      ring: "2px",
+      ringColor: "ui.black",
```

Comment:
This is the first time `ring` and `ringColor` are used in the DS. We have been using `outline` and `outlineColor`. It seems that the difference is that `ringColor` uses the box-shadow for the outline.

We also are not very consistent about this but in many places the box-shadow is set to "none". Did you have a specific reason why ring is used over outline? Otherwise, it might be better to stick with outline for now.